openapi.yaml

```diff
@@ -345,6 +345,32 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/GenericApiResponse"
+  /execAclRequestDecline:
+    post:
+      tags:
+        - ACL
+      operationId: declineAclRequests
+      summary: Decline an ACL request
+      parameters:
+        - name: req_no
+          in: query
+          required: true
+          schema:
+            type: string
+          example: "1008"
+        - name: reasonForDecline
+          in: query
+          required: true
+          schema:
+            type: string
+          example: "I+don%27t+know+why+you+need+access"
```

Comment:
Just for me to understand -> is this encoded for some reason? 🤔

Reply:
Yes, because it's a string picked from a query parameter 😬 So it will be URL encoded

Reply:
I *think* the schema already describes it with `"in": "query"` here: https://github.com/aiven/klaw/pull/567/files#diff-d910ba2ef878f7db0223a966b81c8b3f3b65027bb39e4431bb05140171eece39R362

But I added missing descriptions anyway ^^

Reply:
Ah, yes, the `in query` could have been a hint for me ^^ But it def does not hurt to have an extra description!  Thanks!